_includes/manuals/1.8/3.2.3_installation_scenarios.md

```diff
@@ -168,4 +171,16 @@ foreman-installer \
   --foreman-proxy-oauth-consumer-secret=<secret here>
 {% endhighlight %}
 
-Also ensure here that the dhcp-interface argument is updated for the interface to run DHCP on.
+Also ensure here that the dhcp-interface argument is updated for the interface
+to run DHCP on. After configuration, make sure to create new Subnet (or import
+from existing) in the Foreman interface.
+
+While it is possible to define the same DHCP range in Foreman, it's usually
+good practice to select range from outside of pool defined via the installer,
```

Comment:
`good practice to select a range from outside the pool defined in the installer,`